docs/core/tools/dotnet-sln.md

```diff
@@ -10,10 +10,9 @@ ms.date: 10/29/2019
 <!-- todo: uncomment when all CLI commands are reviewed
 [!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
 -->
-
 ## Name
 
-`dotnet sln` - Modifies a .NET Core solution file.
+`dotnet sln` - Options for the projects in a .NET Core solution file.
```

Comment:
I think the previous wording was actually better here.

Reply:
Wondering if we should break this down into multiple pages like we did with dotnet tool